ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-release-4.18.yaml

```diff
@@ -24,6 +24,9 @@ promotion:
   - disabled: true
     name: "4.18"
     namespace: ocp
+  - disabled: true
```

Comment:
why is this disabled?

Reply:
maybe because master branch promote to 4.18 as well, so disable this to avoid conflits